### ISSUE TITLE
Make explainer video about the project

### ISSUE THREAD

**Issue by Ainali** [govdirectory/website#38]
Make a short video, probably shorter than 3 minutes that informes the vision and mission of the project, summarizes what we are currently doing and how people can help.

**Comment by Ainali** [comment]
This is sort of the second half of the code for all video, but I guess if we recorded it separately, it would be even better.

**Comment by RVA2869** [comment]
@Ainali This is already completed right? [Introduction to Govdirectory](https://commons.wikimedia.org/wiki/File:Introduction_to_Govdirectory.webm)

**Comment by Ainali** [comment]
@RVA2869 Correct! I wonder if we should embed this video too (or link it if it's not possible)? It could go either in this README or on the [GitHub organization page](https://github.com/govdirectory), or both even.

**Comment by RVA2869** [comment]
@Ainali Apparently Github does not allow the use of iframe in the readme file. (For security reasons.)

Alternative way:
`[![Video](<--Insert a thumbnail image here-->)](https://upload.wikimedia.org/wikipedia/commons/c/c0/Introduction_to_Govdirectory.webm)
`

**Comment by Ainali** [comment]
I guess a third way would be to upload a mp4 to the repo. What do you think about that @Abbe98 ?
